scss/_utilities.scss

```diff
@@ -87,3 +87,41 @@
 @include text-emphasis-variant('.text-warning', $brand-warning);
 
 @include text-emphasis-variant('.text-danger', $brand-danger);
+
+// Weight
+
+.text-black {
```

Comment:
This name seems confusing since it could easily be misinterpreted as meaning `color: black;`